### PR TITLE
Add Governance page to docs.

### PR DESCRIPTION
**Pull request by anthonyshew**

### CODE REVIEW

File: docs/pages/governance.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Stability and release phases
+
+- **Experimental**: Experimental features lack essential capabilities required to be ready for stability. The feature is considered to be under active development and is being built to transition into Beta.
+- **Beta**: Beta means the feature does not yet meet our quality standards for a stable release. There is also usually a need for more information or feedback from Turbo developers to validate this feature works as expected in a variety of environments.
+- **Stable**: When the features reach stability, it is considered ready for use in production. APIs marked as stable will only change in a backward-compatible way unless there is a major release.
+- **Deprecated**: A feature is in the process of being removed. Deprecated features include documentation for existing users with the next steps and information on when to expect the feature to be removed.
+
+## Turbo Support
```
Comment (tknickman) [inline docs/pages/governance.mdx:39]: Should we mention `vercel.com/solutions/turborepo` here anywhere? It's a great enterprise landing page, but doesn't get enough hooks imo.

**Review by tknickman — APPROVED** [review]

File: docs/pages/governance.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+**Turbopack Lead**: Tobias Koppers
+
+**Core Team**: Turborepo and Turbopack teams at [Vercel](https://vercel.com) manage the direction of Turbo and leadership of the project.
+
+**Documentation**: DevRel teams at [Vercel](https://vercel.com) manage documentation and learning platforms.
+
+## Roadmap and RFC Process
+
+Large architecture decisions and features start as a [Request for Comments (RFC)](https://github.com/vercel/turbo/discussions/categories/rfc) inside GitHub Discussions. The entire Turbo community is invited to be involved in the process and discuss the tradeoffs of proposed work.
```
Comment (mehulkar) [inline docs/pages/governance.mdx:28]:
```suggestion
Large architecture decisions and features start as a [Request for Comments (RFC)](https://github.com/vercel/turbo/discussions/categories/rfc) inside GitHub Discussions. The Turbo community is invited to be involved in the process and discuss the tradeoffs of proposed work.
```

File: docs/pages/governance.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Stability and release phases
+
+- **Experimental**: Experimental features lack essential capabilities required to be ready for stability. The feature is considered to be under active development and is being built to transition into Beta.
+- **Beta**: Beta means the feature does not yet meet our quality standards for a stable release. There is also usually a need for more information or feedback from Turbo developers to validate this feature works as expected in a variety of environments.
+- **Stable**: When the features reach stability, it is considered ready for use in production. APIs marked as stable will only change in a backward-compatible way unless there is a major release.
+- **Deprecated**: A feature is in the process of being removed. Deprecated features include documentation for existing users with the next steps and information on when to expect the feature to be removed.
```
Comment (mehulkar) [inline docs/pages/governance.mdx:37]:
```suggestion
- **Deprecated**: A feature is expected to be removed. Deprecated features should be documented, and provide runtime logs for existing users with the next steps and information on when to expect the feature to be removed.
```

File: docs/pages/governance.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+## Turbo Support
+
+For individual developers looking for free support on their Turbo repositories, they can ask questions in [GitHub Discussions](https://github.com/vercel/turbo/discussions) or in [the Turbo Discord](https://turbo.build/discord).
```
Comment (mehulkar) [inline docs/pages/governance.mdx:41]:
```suggestion
Developers looking for community support on their Turbo repositories can ask questions in [GitHub Discussions](https://github.com/vercel/turbo/discussions) or in [the Turbo Discord](https://turbo.build/discord).
```

File: docs/pages/governance.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+For companies looking for paid support on their Turbo repositories, they can [contact the Turbo team at Vercel](https://vercel.com/contact) for more information.
```
Comment (mehulkar) [inline docs/pages/governance.mdx:43]:
```suggestion
Companies looking for paid support can [contact the Turbo team at Vercel](https://vercel.com/contact) for more information.
```

**Pull request merged**